docs/querying/functions.md

```diff
@@ -90,12 +79,15 @@ the nearest integer value greater than or equal to v.
 
 For each input time series, `changes(v range-vector)` returns the number of
 times its value has changed within the provided time range as an instant
-vector.
+vector. A float sample followed by a histogram sample, or vice versa, counts as
+a change. A counter histogram sample followed by a gauge histogram sample with
+otherwise exactly the same values, or vice versa, does not count as a change.
 
 ## `clamp()`
 
-`clamp(v instant-vector, min scalar, max scalar)`
-clamps the sample values of all elements in `v` to have a lower limit of `min` and an upper limit of `max`.
+`clamp(v instant-vector, min scalar, max scalar)` clamps the values of all
+float samples in `v` to have a lower limit of `min` and an upper limit of
+`max`. Histogram samples in the input vector are ignored silently.
 
 Special cases:
```

Comment:
The result is also empty in the special cases, so maybe say that special cases for float series?

Reply:
I have amended the NaN limit special case to only apply to float samples.